src/subscription_manager/managercli.py

```diff
@@ -2322,6 +2327,7 @@ def _do_command(self):
                                 data['suggested'],
                                 data['service_level'] or "",
                                 data['service_type'] or "",
+                                subNumber,
```

Comment:
could this be
```
data['subscriptionId'] or "",
```
Without the bits around line 2312-2314?

I think 'data' here should always be populated with a subscriptionsId key in get_available_entitlements?